config/webpack.config.js

```diff
@@ -207,44 +204,17 @@ const configGenerator = (options) => {
     if (options.buildtype === 'production') {
       sourceMap = 'https://s3-us-gov-west-1.amazonaws.com/www.vets.gov';
     }
+
     baseConfig.plugins.push(new webpack.SourceMapDevToolPlugin({
       append: `\n//# sourceMappingURL=${sourceMap}/generated/[url]`,
       filename: '[file].map',
     }));
-    baseConfig.module.rules.push({
-      test: /debug\/PopulateVeteranButton/,
-      use: {
-        loader: 'null-loader'
-      }
-    });
-    baseConfig.module.rules.push({
-      test: /debug\/PerfPanel/,
-      use: {
-        loader: 'null-loader'
-      }
-    });
-    baseConfig.module.rules.push({
-      test: /debug\/RoutesDropdown/,
-      use: {
-        loader: 'null-loader'
-      }
-    });
 
-    baseConfig.plugins.push(new WebpackMd5Hash());
+    baseConfig.plugins.push(new webpack.HashedModuleIdsPlugin());
     baseConfig.plugins.push(new ManifestPlugin({
       fileName: 'file-manifest.json'
     }));
-    baseConfig.plugins.push(new ChunkManifestPlugin({
```

Comment:
This isn't necessary now since the runtime is not inserted into the vendor chunk.